frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ReleasePlanMilestoneItem/milestoneStatusUtils.ts

```diff
@@ -10,6 +10,10 @@ export const calculateMilestoneStatus = (
     environmentIsDisabled: boolean | undefined,
     allMilestones: IReleasePlanMilestone[],
 ): MilestoneStatus => {
+    if (milestone.pausedAt) {
+        return { type: 'paused' };
+    }
+
```

Comment:
This is the main key, align what paused status means. Now if milestone progression is paused, then milestone is also.